src/components/notification-box/Notification.tsx

```diff
@@ -0,0 +1,21 @@
+import Image from "next/image";
+import Link from "next/link";
+
+type Props = {
+  title: string;
+  body: string;
+  redirectTo: string;
+  Icon: string;
```

Comment:
I'm thinking why "Icon" is started with capital letter. Typo?

Reply:
well i followed your naming of imported icons

Reply:
we will probably change it when creating icons as components later or using package